tools/local-storage/src/storage.ts

```diff
@@ -0,0 +1,29 @@
+import storage from 'local-storage-fallback'
+
+export function get(key: string): string | null {
+  return storage.getItem(`chainlink.${key}`)
```

Comment:
This is all happening client-side, so we don't have to worry about injection attacks, right?

Reply:
Yeah, client side https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage

You can change these values in the browser. E.g. set `authorized = true` to get access to the templates. But the API requests will give you a `401` and you'll be redirected.